acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/SignerConfigurationBuilder.java

```diff
@@ -196,11 +197,16 @@ public SignerConfigurationBuilder withUseConfigFile(final boolean useConfigFile)
     return this;
   }
 
-  public SignerConfigurationBuilder withAltairForkEpoch(final long altairForkEpoch) {
+  public SignerConfigurationBuilder withAltairForkEpoch(final Optional<Long> altairForkEpoch) {
```

Comment:
I could've used `Optional.ofNullable()` .... but most of the time calling this method, we would use literal values like `0L` ... I wanted to force `Optional.of` .. or `Optional.empty` instead of passing `null`.

Reply:
It's ultimate usage is in DSL where we construct command line arguments to create a web3signer instance.

Reply:
Doesn't the `Builder.with...` method imply it's an optional argument already? I would only expect to call this method if I had a non-null value to pass in.

Reply:
fair point :)

Reply:
suggestion incorporated!